CMakeLists.txt

```diff
@@ -91,6 +91,17 @@ endif()
 
 include_directories(${CMAKE_BINARY_DIR}) # Make the exv_conf.h file visible for the full project
 
+if(CMAKE_CXX_STANDARD GREATER_EQUAL 20)
+  include(CheckIncludeFileCXX)
+  check_include_file_cxx(format HAVE_STD_FORMAT)
+else()
+  set(HAVE_STD_FORMAT FALSE)
+endif()
```

Comment:
We can also (like meson) simplify this to
```
include(CheckCXXSymbolExists)
check_cxx_symbol_exists(std::format "format" HAVE_STD_FORMAT)
```

Reply:
That's what I do in meson, yes.

Reply:
Done.